fairlearn/reductions/moments/conditional_opportunity.py

```diff
@@ -27,15 +54,16 @@ def gamma(self, predictor):
         self.tags["pred"] = pred
         expect_grp = self.tags.groupby("grp").mean()
         expect_attr_grp = self.tags.groupby(
-            ["grp", "protected_attribute"]).mean()
+            ["grp", _GROUP_ID]).mean()
         expect_attr_grp["diff"] = expect_attr_grp["pred"] - expect_grp["pred"]
         g_unsigned = expect_attr_grp["diff"]
         g_signed = pd.concat([g_unsigned, -g_unsigned],
                              keys=["+", "-"],
-                             names=["sign", "grp", "protected_attribute"])
+                             names=["sign", "grp", _GROUP_ID])
```

Comment:
Making `sign` and `grp` into constants as well would be nice

Reply:
Yes that's the idea... I just ran out of time on train and wanted to get this PR out for review.